install/kubernetes/helm/istio/values.yaml

```diff
@@ -13,7 +13,13 @@
 #
 gateways:
   enabled: true
-
+  istio-ilbgateway: 
+    enabled: false
+    serviceAnnotations:
+      cloud.google.com/load-balancer-type: "internal"
+      #  service.beta.kubernetes.io/alicloud-loadbalancer-address-type: "intranet"
```

Comment:
I am not sure we want these in the top level values file. We try to keep this one small (just `enabled`) and have the settings come from the chart (gateways/values.yaml)